javascript/statements.json

```diff
@@ -796,6 +796,72 @@
               "deprecated": false
             }
           }
+        },
+        "namespace": {
+          "__compat": {
+            "description": "<code>export * as ns</code>",
+            "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Statements/export",
+            "spec_url": "https://tc39.es/ecma262/#sec-exports",
+            "support": {
+              "chrome": {
+                "version_added": "72"
+              },
+              "chrome_android": {
+                "version_added": "72"
+              },
+              "edge": {
+                "version_added": "79"
+              },
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": {
+                "version_added": false
+              },
+              "ie": {
+                "version_added": false
+              },
+              "nodejs": [
```

Comment:
As a subfeature of `export`, we won't need the flags data here. It can be just be `"version_added": "12.0.0"`